src/kibana/plugins/dashboard/services/saved_dashboards.js

```diff
@@ -36,8 +36,9 @@ define(function (require) {
     };
 
 
-    this.find = function (searchString) {
+    this.find = function (searchString, size) {
       var self = this;
+      size = size || 100;
```

Comment:
Hmm, I guess technically it's possible to pass in `0` as the size here (which could be valid, if you were just doing `this.find` to end up getting the `total` back (and no docs). Maybe doing a check for `undefined` or our standard `!= null` might be better here...